spark/v3.3/spark/src/main/java/org/apache/iceberg/spark/procedures/AddFilesProcedure.java

```diff
@@ -120,9 +121,16 @@ public InternalRow[] call(InternalRow args) {
       checkDuplicateFiles = args.getBoolean(3);
     }
 
-    long addedFilesCount =
-        importToIceberg(tableIdent, sourceIdent, partitionFilter, checkDuplicateFiles);
-    return new InternalRow[] {newInternalRow(addedFilesCount)};
+    return importToIceberg(tableIdent, sourceIdent, partitionFilter, checkDuplicateFiles);
+  }
+
+  private InternalRow[] toOutputRows(Snapshot snapshot) {
+    return new InternalRow[] {
```

Comment:
I think you can beat the auto formatting messing up line 132 if you save snapshot.summary() to a temp var like summary

Reply:
or static import the SnapshotSummary Constants

Reply:
thank you @RussellSpitzer , now they looks better now.

Reply:
looks like checkstyle does not like my static import of member, I'll have a temp var for summary 

```
> Task :iceberg-spark:iceberg-spark-3.3_2.12:checkstyleMain
[ant:checkstyle] [ERROR] /Users/stevezhang/workspace/iceberg/spark/v3.3/spark/src/main/java/org/apache/iceberg/spark/procedures/AddFilesProcedure.java:21:49: Using a static member import should be avoided - org.apache.iceberg.SnapshotSummary.ADDED_FILES_PROP. [AvoidStaticImport]
[ant:checkstyle] [ERROR] /Users/stevezhang/workspace/iceberg/spark/v3.3/spark/src/main/java/org/apache/iceberg/spark/procedures/AddFilesProcedure.java:22:49: Using a static member import should be avoided - org.apache.iceberg.SnapshotSummary.CHANGED_PARTITION_COUNT_PROP. [AvoidStaticImport]

```